static/src/javascripts/projects/common/modules/commercial/third-party-tags/plista.js

```diff
@@ -0,0 +1,87 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/config',
+    'common/utils/mediator',
+    'common/utils/detect',
+    'common/utils/template',
+    'common/modules/identity/api',
+    'text!common/views/commercial/plista.html',
+    'lodash/collections/contains'
+], function (fastdom,
+             $,
+             config,
+             mediator,
+             detect,
+             template,
+             identity,
+             plistaTpl,
+             contains) {
+    var containerOuterClassName = 'js-plista',
+        containerInnerClassName = 'js-plista-container';
+
+    return {
+        load: function () {
+            var $containerOuter = $('.' + containerOuterClassName),
+                $containerInner = $('.' + containerInnerClassName);
+
+            fastdom.write(function () {
+                $containerOuter.css('display', 'block');
+                $containerInner.append($.create(template(plistaTpl)));
+
+                this.embed('462925f4f131001fd974bebe', 'innerArticle', 'au', undefined, undefined, containerInnerClassName);
+            }.bind(this));
+        },
+
+        // a modification of the code provided by Plista; altered to be a lazy load rather than during DOM construction
+        embed: function (p, w, g, u, c, containerTag) {
```

Comment:
If we're writing this ourselves, that means we'll be maintaining it ourselves, so we should probably use the naming conventions we'd use elsewhere in our own code, rather than trying to resemble the widget code plista gave us (which will only diverge further from our implementation over time).